README.md

```diff
@@ -198,6 +198,7 @@ Safe Logging can be found at [github.com/palantir/safe-logging](https://github.c
 - `UnnecessarilyQualified`: Types should not be qualified if they are also imported.
 - `DeprecatedGuavaObjects`: `com.google.common.base.Objects` has been obviated by `java.util.Objects`.
 - `JavaTimeSystemDefaultTimeZone`: Avoid using the system default time zone.
+- `PreventUsingIncubatingMethods`: Prevents calling Conjure incubating APIs unless you explicitly opt-out of the check on a per-use or per-project basis.
```

Comment:
Thoughts on renaming this to something like `IncubatingConjureEndpoint`? We haven't been great about naming, but most error-prone checks are named to describe the bad thing they detect.

Reply:
That's fine. I don't think we have code using the suppression yet, so no breakages.